meteor/server/cronjobs.ts

```diff
@@ -46,79 +39,21 @@ export function nightlyCronjobInner(): void {
 	logger.info('Nightly cronjob: starting...')
 	const system = getCoreSystem()
 
-	// Clean up Rundown data cache:
-	// Remove caches not related to rundowns:
-	let rundownCacheCount = 0
-	const rundownIds = _.map(Rundowns.find().fetch(), (rundown) => rundown._id)
-	IngestDataCache.find({
-		rundownId: { $nin: rundownIds },
-	}).forEach((roc) => {
-		lowPrioFcn(() => {
-			IngestDataCache.remove(roc._id)
-		})
-		rundownCacheCount++
-	})
-	if (rundownCacheCount) logger.info('Cronjob: Will remove cached data from ' + rundownCacheCount + ' rundowns')
-
-	const cleanLimitTime = getCurrentTime() - 1000 * 3600 * 24 * 50 // 50 days ago
-
-	// Remove old PartInstances and PieceInstances:
```

Comment:
I don't see this cleanup being done anymore

Reply:
I commented on where PartInstances are handled, let's continue that talk there.
I also added a cleanup of PieceInstances when their PartInstance has been removed [in cleanup.ts:292](https://github.com/nrkno/sofie-core/pull/904/files#diff-975acd74a53b0f96f509165792e86422ec38d8142023efe9d9ca757d8c6381ffR292).